sdk-commons/src/main/java/org/openmrs/maven/plugins/model/DistroProperties.java

```diff
@@ -291,4 +291,31 @@ private boolean hasPlaceholder(Object object){
         int index = asString.indexOf("{");
         return index != -1 && asString.substring(index).contains("}");
     }
+
+    public boolean contains(String propertyName) {
+        return properties.containsKey(propertyName);
+    }
+
+	/**
+	 * Adds a dependency with the specified version to the properties.
+	 *
+	 * @param dependency The name of the dependency.
+	 * @param version The version of the dependency.
+	 */
+	public void add(String dependency, String version) {
+		if (StringUtils.isBlank(dependency) || StringUtils.isBlank(version)) {
+			log.error("Dependency name or version cannot be blank");
+			return;
+		}
+		
+		if (dependency.startsWith("omod.") || dependency.startsWith("owa.") || dependency.startsWith("war")
+		        || dependency.startsWith("spa.frontendModule")) {
+			properties.setProperty(dependency, version);
+			log.info("Added dependency: {} with version: {}", dependency, version);
+		}
+	}
+	
+	public String get(String contentDependencyKey) {
+		return properties.getProperty(contentDependencyKey);
+	}
 }
```

Comment:
This file is mostly indented with spaces, but this change adds tabs for this section only. It's preferable to be consistent about this.